docs/collaboration.md

```diff
@@ -0,0 +1,113 @@
+Collaborations
+==============
+
+Collaborations are used to share folders between users or groups. They also
+define what permissions a user has for a folder.
+
+
+Add a Collaboration
+-------------------
+
+You can add a collaboration on a folder or a file by calling `item.collaborate(accessible_by, role, can_view_path=None, notify=None, fields=None)`. The
+`role` parameter determines what permissions the collaborator will have on the folder.
+
+```python
+from boxsdk.object.collaboration import CollaborationRole
+folder_id = '1234'
+user_id = '1111'
+collaborator = client.user(user_id)
+collaboration = client.folder(folder_id).collaborate(collaborator, CollaborationRole.VIEWER)
+```
+
+Alternatively you can also invite a user with their email address.
+
+```python
+from boxsdk.object.collaboration import CollaborationRole
+folder_id = '1234'
+email_of_invitee = "testuser@example.com"
+collaboration = client.folder(folder_id).collaborate_with_login(email_of_invitee, CollaborationRole.VIEWER)
+```
+
+Edit a Collaboration
+--------------------
+
+A collaboration can be edited by using `collaboration.update_info()`.
+
+```python
+from boxsdk.object.collaboration import CollaborationRole
+collaboration_id = '1111'
+collaboration_update = {'role': CollaborationRole.EDITOR}
+updated_collaboration = client.collaboration(collaboration_id).update_info(collaboration_update)
```

Comment:
This example doesn't use the overridden version of this method that takes explicit `status` and `role` params; should we update that method and use it in these examples?